src/testE2E/amazonq/featureDev.test.ts

```diff
@@ -0,0 +1,127 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as got from 'got'
```

Comment:
Not important for this PR, but for browser compatibility we are using the `fetch` implementation in `src/common/request.ts` instead of `got` (doesn't work in browser).

Just something to keep in mind going forward.